src/git-duet/git_config.go

```diff
@@ -114,6 +122,33 @@ func (gc *GitConfig) RotateAuthor() (err error) {
 	return nil
 }
 
+func makeRotatedTeam(teamAuthor *Pair, teamCommitter *Pair) (newAuthor *Pair, newCommitter *Pair) {
+	authorInitials := teamAuthor.Initials
+	authorName := teamAuthor.Name
+	authorEmail := teamAuthor.Email
+	committerInitialsList := strings.Split(teamCommitter.Initials, ", ")
+	committerNamesList := strings.Split(teamCommitter.Name, ", ")
+	committerEmailsList := strings.Split(teamCommitter.Email, ", ")
+	InitialsList := append(committerInitialsList, authorInitials)
```

Comment:
I am not sure there is precedent for using a capitalized variable name local to a function like this.  I have never seen it done before